# This patch changes Vulkan Memory Allocator to declare its version without using `VK_MAKE_VERSION()`.
# Otherwise, we get a compilation error in vk_mem_alloc_wrapper.cpp in ANGLE.

--- a/include/vk_mem_alloc.h
+++ b/include/vk_mem_alloc.h
@@ -135,7 +135,7 @@ extern "C" {
     #endif
 #endif
 
-#define VMA_VERSION (VK_MAKE_VERSION(3, 4, 0))
+#define VMA_VERSION (3 << 22 | 4 << 12 | 0 << 0)
 
 #if !defined(VMA_VULKAN_VERSION)
     #if defined(VK_VERSION_1_4)
